Makefile

```diff
@@ -1,7 +1,8 @@
-.PHONY: syntax-check
+.PHONY: syntax-check pre-commit-check print-rollouts
```

Comment:
Hum, why is this adding `pre-commit-check` here?

Reply:
based on the review comment, I added the pre-commit check to the syntax check here.